server/scripts/create_database.sql

```diff
@@ -18,7 +18,6 @@ CREATE TABLE clients
     residency_status INT,
     country_of_origin VARCHAR(120), 
     time_in_scotland VARCHAR(120),
-    language_spoken VARCHAR(120),
```

Comment:
It looks like this was deleted by mistake—let's add it back in

Reply:
Thanks Coung is alredy added back